docker-app/qfieldcloud/core/admin.py

```diff
@@ -1072,18 +1159,26 @@ class OrganizationAdmin(QFieldCloudModelAdmin):
 
     list_select_related = ("organization_owner",)
 
-    list_filter = ("date_joined",)
+    list_filter = ("date_joined", ActiveUsersFilter)
 
     autocomplete_fields = ("organization_owner",)
 
-    @admin.display(description=_("Active users (last billing period)"))
-    def active_users(self, instance) -> int | None:
-        # The relation 'current_subscription_vw' is not instantiated unless the organization
-        # does have a current subscription
-        if hasattr(instance, "current_subscription_vw"):
-            return instance.current_subscription_vw.active_users_count
-        else:
-            return None
```

Comment:
That one seemed correct. Although I would use (not tested):

```
instance.useraccount.current_subscription().active_users_count()
```

Reply:
Are you commenting code of this PR or of master? Is there an issue with the code of this PR?

Reply:
... And then add a similar method in AbstractSubscription for the `list_active_users_jobs_deltas_count` , where  you can again maybe create some common base method (not sure though) to query the requested time range.

(Just in the end those two methods should be traceable  in the code as being closely related IMO.)

Reply:
Ah sorry for confusion. No this one above (master) seems right.

Reply:
Which problem have you run into with the one offered by the PR?

Reply:
No, it is good that part. Sorry. But we should implement the `list_active_users_jobs_deltas_count` just the same way.

Reply:
I think it would make it _much_ easier for me if you layered out your review into something like:
1) is the code correct (nothing broken?)
2) is the code complete (are all the target features there?)
3) is the code efficient and maintainable (DRY, SOLID, etc.)
4) does the code meet all the other conventions agreed upon by the maintainters of the codebase?
It's a bit difficult for me to go back and forth between all these aspects, and to keep asking questions about which aspect this or that comment is about, because in many cases it's just not clear. 

It should be possible to proceed with "rounds" (all comments about one layer before moving on to the next layer) or, if not appropriate, to just clarifiy what aspect a comment is about.

Reply:
@why-not-try-calmer yes agree with your suggestions - like this structure. Was a bit an unlucky history in this specific PR because I started with broken local stack but still found first corrections and wanted to post them asap. Then I came up with many more and you were already processing the first one. 

Further I was not very much into this part of codebase neither and found myself misunderstanding a couple of times. It is quite a complex task ...

I appreciate your input and will try to stay structured the next time.